auth-api/migrations/versions/31ec16d4f1e9_product_migration_for_vs.py

```diff
@@ -0,0 +1,49 @@
+"""product_migration_for_vs
+
+Revision ID: 31ec16d4f1e9
+Revises: a37f90e6802d
+Create Date: 2021-04-30 10:01:10.794355
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from alembic import op
+from flask import current_app
+
+from auth_api.models import db
+from auth_api.services.products import Product
+from auth_api.services.rest_service import RestService
+
+
+# revision identifiers, used by Alembic.
+revision = '31ec16d4f1e9'
+down_revision = 'a37f90e6802d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Query all orgs which are linked to BCOL.
+    conn = op.get_bind()
+    org_res = conn.execute(
+        "select o.id, o.bcol_user_id from orgs o where bcol_user_id is not null and bcol_account_id is not null;"
```

Comment:
may be check the status_code as well ;there could be orgs which are inactive... its ok otherwise as well...